tests/patterns/scala/metavars_in_interpolated_strings.scala

```diff
@@ -0,0 +1,5 @@
+//ERROR: match
```

Comment:
It would be nice to have a taint test (in `tests/rules/taint_interpolated_strings.scala` or sth) as well.